tmpl/partials/footer.html

```diff
@@ -27,6 +27,7 @@
               <!-- <a href="//ipfs.io/ecosystem">Ecosystem</a><br /> -->
               <!-- <a href="//ipfs.io/roadmap">Roadmap</a><br /> -->
               <a href="//ipfs.io/blog">Blog</a><br />
+              <a href="https://docs.google.com/forms/d/1fXchtQEQ6WDjedSk46ZZiye8vl4UxvyRZiltSOv2huc/viewform">DMCA</a><br />
```

Comment:
This should not be in the footer of all https://ipfs.io pages. i just checked many very popular media sites and not one has the text "dmca" anywhere on their normal pages, the most they have is "Help" or "Support" which lead to help centers that answer the question, and only sometimes "Legal".

- we already need a "Legal" page under "Protocol" that goes to http://ipn.io/legal/ -- we can put a link on that page.
- we can also have a page describing the ipfs public gateway service in general -- maybe at https://ipfs.io/docs/public-gateway that links to a ToS + Privacy Policy. Maybe have a link to the the form there.